### DISCUSSION TITLE
不支援 emoji 嗎？

### DISCUSSION THREAD

**Discussion by charleslcso** [cloudreve/cloudreve#1497] [Q&A] [answered]
![Screen Shot 2022-10-08 at 5 18 03 PM](https://user-images.githubusercontent.com/520537/194700131-30e5a38b-5d17-4692-ad04-7c5ec3233bef.png)

不支援 emoji 嗎？

**Comment by topjohncian** [comment, marked as answer]
数据库格式可使用 `utf8mb4`
